packages/state-manager/src/sagas/socket/startConnection/startConnection.saga.ts

```diff
@@ -194,7 +194,13 @@ export function subscribe(socket: Socket) {
     })
     // Errors
     socket.on(SocketActionTypes.ERROR, (payload: ErrorPayload) => {
-      log(payload)
+      // FIXME: It doesn't look like log errors have the red error
+      // color in the console, which makes them difficult to find.
+      // Also when only printing the payload, the full trace is not
+      // available.
+      log.error(payload)
+      console.error(payload)
+      console.error(payload.trace)
```

Comment:
I'm pretty convinced you can use a single log
```
console.error(payload, payload.trace)
```
or
```
console.error({...payload})
```
Although, I'm not 100% sure whether we really want to use `console.error` here. As far as I understand it only helps us read the logs while debugging some specific cases. Those errors we display in the UI anyway.